embeddings/get_started_with_embedding_tuning.ipynb

```diff
@@ -1,1438 +1,1438 @@
+        "id": "eldc535Y7xmD"
+      },
+      "outputs": [],
+      "source": [
+        "metric_df = get_metrics(job)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "W-s_AEuoaKSd"
+      },
+      "outputs": [],
+      "source": [
+        "metric_df.to_dict()"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "qMBUmPFgBRWi"
+      },
+      "outputs": [],
+      "source": [
+        "metric_df"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ZP6nCnJ2_6wU"
+      },
+      "source": [
+        "### Deploy the embedding tuned model on Vertex AI Prediction\n",
+        "\n",
+        "To deploy the embedding tuned model, you need to create an Vertex AI Endpoint.\n",
+        "\n",
+        "Then you deploy the tuned embeddings model to the endpoint."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "d5LtEGEbAHPd"
+      },
+      "source": [
+        "#### Create the endpoint"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Q2KRaRzHAF8f"
+      },
+      "outputs": [],
+      "source": [
+        "endpoint = aiplatform.Endpoint.create(\n",
+        "    display_name=\"tuned_custom_embedding_endpoint\",\n",
+        "    description=\"Endpoint for tuned model embeddings.\",\n",
+        "    project=PROJECT_ID,\n",
+        "    location=REGION,\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "xGLjbFY-AYi3"
+      },
+      "source": [
+        "#### Deploy the tuned model"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "mdfedMEj1ZNy"
+      },
+      "source": [
+        "Get the tuned model."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "ndE5WGVkBQA2"
+      },
+      "outputs": [],
+      "source": [
+        "model = get_uploaded_model(job)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "CqkAAzvV1ewQ"
+      },
+      "source": [
+        "Deploy the tuned model to the endpoint."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "nOZyyPALDcSP"
+      },
+      "outputs": [],
+      "source": [
+        "endpoint.deploy(\n",
+        "    model,\n",
+        "    accelerator_type=PREDICTION_ACCELERATOR_TYPE,\n",
+        "    accelerator_count=PREDICTION_ACCELERATOR_COUNT,\n",
+        "    machine_type=PREDICTION_MACHINE_TYPE,\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8VVac8ah2DzJ"
+      },
+      "source": [
+        "### Retrieve similar items using the tuned embedding model\n",
```

Comment:
This section should compare the results before and after tuning for the user, and show noticeably better results after tuning.

Reply:
In the Evaluate the tuned model section, the user will see an improvements in the ranking metrics of the embeddings model. And in the retrieve similar items using the tuned embedding model you can explore how better are the results looking at the associated dataframe.